.idea/SaaSquatch-HubSpot-integration.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these files for mocha?

Reply:
I think it's project settings file made by WebStorm. I will delete it.